### ISSUE TITLE
Principle(s) about identity on the Web?

### ISSUE THREAD

**Issue by rhiaro** [w3ctag/design-principles#324]
Architectural principles around identity and identifiers on the Web [have been proposed on the mailing list](https://lists.w3.org/Archives/Public/www-tag/2021Jun/0000.html). There's discussion on the list; creating this issue to track.

So far we've discussed whether we could write something in Design Principles about this, or if it should be a standalone document (finding?), or if it actually may be too early (since this is a broad and rapidly evolving domain) for the TAG to be weighing in authoritatively.

**Comment by sheldrake** [comment]
The definition of concepts and terms is always a good place to start of course.

[Brubaker and Cooper (2000)](https://voidnetwork.gr/wp-content/uploads/2016/09/Beyond-Identity-by-Rogers-Brubaker-and-Frederick-Cooper.pdf) argue that "identity" is in crisis, a devaluation of its meaning caused by a proliferation of varying conceptualizations and applications.

Nevertheless, ethically speaking, all conceptualizations are relevant here if only because:

1. Each conceptualization exists because at least one discipline finds it useful in understanding the world, our communities, and ourselves;
2. Working with one or a few conceptualizations has consequences for the system parameters and qualities described by the others;
3. There is no such thing as 'digital identity' in the context of humans, only digitally mediated and augmented human identity; and
4. Many millions of people have been excluded, persecuted, and murdered with the assistance of prior identity architectures, and no other facet of information technology smashes into the human condition in quite the same way.

While getting familiar with multiple conceptualizations of identity is impractical for many, we've found at the AKASHA Foundation that considering conceptualizations in terms of being noun-like or verb-like helps develop a rapid and appropriate feel for the design space. The former are unchanging by design, typified by the comparatively recent bureaucratization of identity (e.g. legal identity). The latter are dynamic, more inter-networked, more natural, and potent with greater complexity (e.g. narrative selves).

Designs for the noun-like typically strive for convenience and/or process efficiency (e.g. collecting taxes; hiring a car). Those attracted to design for the verb-like typically seek to respect and work with the nature of things, including human nature in this context of course. ("If one way be better than another, that you may be sure is nature's way." Aristotle.)

We use the term generative identity to describe approaching 'digital identity' specifically for psychological, sociological, and ecological health.

We endorse [Jonathan Donner's recommendation](https://medium.com/caribou-digital/the-difference-between-digital-identity-identification-and-id-41580bbb7563) to distinguish the terms "identity", "identification", and "ID".

- Identity implies a kind of multidimensional social location of an individual relative to other people and institutions around him or her.
- Identification is a claims verification process.
- ID is an artifact, traditionally tangible, that supports a claim or signals that identification might be possible.

In other words, identity ≠ identification ≠ ID.

To your question @rhiaro — Should the TAG be weighing in authoritatively? Yes imho.

- This is playing out right now, so if not now then never.
- This is where the web was always going.
- This undoubtedly entails principles of Web architecture and a need for coordination.
- The TAG has no commercial agenda here, setting it apart from many of the other 'digital identity' initiatives.

**Comment by agnosis-be** [comment]
On the mailing list the [example of Clark Kent and Superman](https://lists.w3.org/Archives/Public/www-tag/2021Jun/0027.html) highlights the need for a definition of identity. While I lack such a definition, I would like to remind of a down-to-earth approach, that law enforcement would adopt, if they had to deal with Clark Kent and Superman.

On encountering Clark Kent, they would create a record in their descriptions table with first name: Clark, last name: Kent, wears glasses: yes, as well as a record in their identities table (without any descriptive attributes), whereby the former record refers to the latter.

On encountering Superman - not knowing that Superman and Clark Kent are the same person - they would do the same: create a new description record describing Superman as well as a new identity record (likewise without any descriptive attributes) and let the description record point to the newly created identity record.

Once they find out, that Superman and Clark Kent are in fact the same person, they would _keep_ both description records and make sure that _both_ refer to the _same_ identity record (which exists only once).

The rationale behind this is obvious: while it is crucial to store the information that Superman and Clark Kent are the same person, it is equally important to separate the description of Superman from that of Clark Kent. Merging the descriptions would lead to assertions such as: Superman wears glasses, which would be wrong. In addition, with this approach you may express a relationship between one description record and _another_ record (in a third table), such as: Clark Kent rides a bicycle, without asserting that Superman does the same.

**Comment by sheldrake** [comment]
> "I would like to remind of a down-to-earth approach, that law enforcement would adopt."

It's as if dystopian novelists were wasting their time, despite bountiful evidence to the contrary.

**Comment by agropper** [comment]
See https://github.com/w3c-ccg/community/issues/195#issuecomment-888707378 for a human rights perspective.

**Comment by wip-abramson** [comment]
I finally got around to reading this blogpost @sheldrake and much of the literature surrounding identification for development over the last couple of weeks.

I have to say I fully endorse the framing of identification systems and the terminology @sheldrake is pointing to. Identification systems existed before digital identity solutions, and generally create asymmetric power differentials between the entity being identified and the entity performing the identification typically on behalf of some insitution. I like that this terminology makes this difficult to ignore. I also appreciate the distinction between complex human identity and the static attributes collected for the purposes of identification.

Digital identity originated, at least in my view, from the functional requirement for information systems to identify and virtually represent entities executing actions within digital environments. However, today technological artifacts are increasingly being used to augment identification processes applied to real human beings as they navigate the physical world. Making the design of identification systems that ensure the protection of individuals identified from the potential abuses of those performing identifications of vital importance. Unfortunately, there are many examples of technological solutions for identification doing the exact opposite.

I am not saying that identity is not manifest on the web, but rather digital identity more accurately encapsulates a set of technologies, standards and infrastructure to support and facilitate digital identification.

Anyway, I will be using this language throughout my thesis and have found it extremely helpful in how it directs focus.

**Comment by martinthomson** [comment]
Given that we merged #396, can we close this one?
